cdk/src/open-data-platform/frontend/frontend-stack.ts

```diff
@@ -45,5 +51,35 @@ export class FrontendStack extends Stack {
       distribution: this.distribution,
       distributionPaths: ['/*'],
     });
+
+    // Add the tileserver to the Cloudfront distribution to make it publicly available.
+    // TODO: consider splitting this out into another file for organization.
```

Comment:
+1 to splitting the distribution and this out and adding this to props to pass to both separately in the future.

Reply:
ACK.